easybuild/tools/github.py

```diff
@@ -1004,7 +1005,7 @@ def merge_pr(pr):
 
     # also fetch comments
     comments_url = lambda g: g.repos[pr_target_account][pr_target_repo].issues[pr].comments
-    status, comments_data = github_api_get_request(comments_url, github_user)
+    status, comments_data = github_api_get_request(comments_url, github_user, per_page=GITHUB_MAX_PER_PAGE)
```

Comment:
Add a comment here stating that a PR with lots of comments (`> GITHUB_MAX_PER_PAGE `) might fail?